scripts/qa_crowbarsetup.sh

```diff
@@ -2008,7 +2008,9 @@ function reboot_nodes_via_ipmi()
                     ipmitool -H $ip -U root -P $pw power on
                     sleep $((10 + RANDOM % 15))
                 fi
-                ipmitool -H $ip -U root -P $pw power reset) &
+                ipmitool -H $ip -U root -P $pw power off
```

Comment:
does "power cycle" work? in other cases this should probably be moved inside an else block, as you're just powering it on a few lines above, and it can't be healthy to your disks to do that twice in short row.